Header header

float32 rate
float32 fuel_level_main_tank # main tank fuel level
float32 fuel_level_aux_tank # level of additional fuel tank
float32 exhaust_gas_temperature
float32 cylinder_temperature1
float32 cylinder_temperature2
float32 throttle1_angle
float32 throttle2_angle

EngineStateExt relay_state
